e2e-tests/column-mapping/create.hydra

```diff
@@ -0,0 +1,63 @@
+PREFIX api: <https://cube-creator.zazuko.com/vocab#>
+PREFIX hydra: <http://www.w3.org/ns/hydra/core#>
+PREFIX cc: <https://cube-creator.zazuko.com/vocab#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX schema: <http://schema.org/>
+
+HEADERS {
+  x-user "john-doe"
+  x-permission "pipelines:write"
+}
+
+With Class api:EntryPoint {
+    Expect Property cc:projects {
+        With Operation cc:CreateProjectAction {
+            Invoke {
+                Content-Type "text/turtle"
+
+                ```
+                prefix cc: <https://cube-creator.zazuko.com/vocab#>
+                prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+
+                <> cc:projectSourceKind <https://cube-creator.lndo.site/shape/cube-project/create#CSV> ;
+                   rdfs:label "Test Project" ;
+                   cc:namespace <http://example.com/test-cube/> .
+                ```
+            } => {
+                Expect Status 201
+                Expect Link cc:csvMapping {
+                    Expect Property cc:csvSourceCollection {
+                        Expect Operation cc:UploadCSVAction {
+                            Invoke {
+                                Content-Type "text/csv"
+                                Content-Disposition "file; filename=test.jpg"
+
+                                ```
+                                A,B,C
+                                1,2,test
+                                1,2,test
+                                1,2,test1
+                                1,2,test2
+                                .
+                                ```
+                            } => {
+                                Expect Status 201
+                                With Operation cc:CreateColumnMappingAction {
+                                    Invoke {
+                                        Content-Type "application/ld+json"
+
+                                        ```
+                                        ```
+                                    } => {
+                                        Expect Status 201
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
```

Comment:
By using the seed data you will be able to start the test already at the source itself rather than recreating all the way from the top

Use the `ENTRYPOINT` keyword to declare the initial request to make

```suggestion
PREFIX api: <https://cube-creator.zazuko.com/vocab#>
PREFIX hydra: <http://www.w3.org/ns/hydra/core#>
PREFIX cc: <https://cube-creator.zazuko.com/vocab#>
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
PREFIX dcterms: <http://purl.org/dc/terms/>
PREFIX schema: <http://schema.org/>

ENTRYPOINT "cube-project/ubd/csv-source/ubd"

HEADERS {
  x-user "john-doe"
  x-permission "pipelines:write"
}


With Class api:CSVSource {
    Expect Operation cc:CreateColumnMappingAction {
        Invoke {
            Content-Type "application/ld+json"

            ```
            ```
        } => {
            Expect Status 201
        }
    }
}
```